### ISSUE TITLE
[bug]: Subnavigation using `.palette-accent` has wrong colors

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5247]
### 🐞 Describe the bug

The [subnavigation](https://next.design-system.post.ch/?path=/docs/87ceabbb-f552-46eb-8a47-4d84e7f8cef0--docs) component is not accessible when using a dark palette, as the text color does not switch to light.

### 📷 Screenshots

![Image](https://github.com/user-attachments/assets/48ae07ae-57ce-458d-bd5c-5b9589cc49d4)

**Comment by gfellerph** [comment]
I don't think this component is tokenised yet. Therefore, that behaviour is expected.

**Comment by gfellerph** [comment]
On hold until decision to keep or remove subnavigation pattern.

**Comment by alizedebray** [comment]
The subnavigation will be removed in favor of page tabs.
